src/farcasterd/runtime.rs

```diff
@@ -905,6 +911,163 @@ impl Runtime {
                     )?;
                 }
             }
+            Request::FundingInfo(info) => match info {
+                FundingInfo::Bitcoin(BitcoinFundingInfo {
+                    swap_id,
+                    address,
+                    amount,
+                }) if self.auto_fund => {
+                    println!("attempting to auto fund bitcoin address");
+                    use bitcoincore_rpc::{Auth, Client, RpcApi};
+                    use std::env;
+                    use std::path::PathBuf;
+                    use std::str::FromStr;
+
+                    let cookie = env::var("BITCOIN_COOKIE")
+                        .unwrap_or("/home/drgrid/.bitcoin/testnet3/.cookie".into());
```

Comment:
please give pa$$word to your machine so we can use this cookie too ;)

Reply:
(and whatever other cookies are rotting in your jar) 🤫

Reply:
for some reason I can't give `~/.bitcoin` to it.

Reply:
Checkout https://github.com/farcaster-project/farcaster-node/blob/06dd98c2bcdfa05967e0a7754093ec332783aac2/src/opts.rs#L151

Maybe this can do the trick to have it work.

Reply:
That did the trick!